### DISCUSSION TITLE
Show location name instead of "Current Location" in notification

### DISCUSSION THREAD

**Discussion by SpitFire-666** [breezy-weather/breezy-weather#1536] [Q&A] [answered]
### Describe your idea

I just like to be sure that it's updated/updating the correct location, especially when travelling :)

It could say "Suburb, State" or something like that. Cheers

![image](https://github.com/user-attachments/assets/caedd348-6199-4b19-b032-078977b0ae2a)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.3.1 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment, marked as answer]
Use a main source that supports reverse geocoding: https://github.com/breezy-weather/breezy-weather/blob/main/docs/SOURCES.md
